bundles/binding/org.openhab.binding.km200/src/main/java/org/openhab/binding/km200/internal/KM200ErrorService.java

```diff
@@ -0,0 +1,145 @@
+
+/**
+ * Copyright (c) 2010-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.openhab.binding.km200.internal;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The KM200ErrorService representing a error service with its all capabilities
+ *
+ * @author Markus Eckhardt
+ *
+ * @since 1.9.0
+ */
+
+public class KM200ErrorService {
+
+    private static final Logger logger = LoggerFactory.getLogger(KM200ErrorService.class);
+
+    protected Integer activeError = 1;
+
+    /* List for all errors */
+    ArrayList<HashMap<String, String>> errorMap = null;
+
+    KM200ErrorService() {
+        errorMap = new ArrayList<HashMap<String, String>>();
+    }
+
+    /**
+     * This function removes all errors from the list
+     *
+     */
+    void removeAllErrors() {
+        synchronized (errorMap) {
+            if (errorMap != null) {
+                errorMap.clear();
+            }
+        }
+    }
+
+    /**
+     * This function updates the errors
+     *
+     */
+    void updateErrors(JSONObject nodeRoot) {
+        synchronized (errorMap) {
+            /* Update the list of errors */
+            try {
+                removeAllErrors();
+                JSONArray sPoints = nodeRoot.getJSONArray("values");
+                for (int i = 0; i < sPoints.length(); i++) {
+                    JSONObject subJSON = sPoints.getJSONObject(i);
+                    HashMap<String, String> valMap = new HashMap<String, String>();
+                    Map<String, Object> oMap = subJSON.toMap();
+                    for (String para : oMap.keySet()) {
+                        logger.debug("Set: {} val: {}", para, oMap.get(para).toString());
+                        valMap.put(para, oMap.get(para).toString());
+                    }
+                    errorMap.add(valMap);
+                }
+            } catch (Exception e) {
+                logger.error("Error in parsing of the errorlist: {} {}", e.getMessage());
```

Comment:
You have two sets of curly braces here, but only one parameter being passed to them.